core/mapmutex/mapmutex.go

```diff
@@ -2,8 +2,9 @@ package mapmutex
 
 import (
 	"fmt"
-	"github.com/LK4d4/trylock"
 	"sync"
+
+	"github.com/LK4d4/trylock"
```

Comment:
Think we should remove this dep actually?

See: LK4D4/trylock#3

Reply:
okay, i've decided we should address this in a different pr (only bc this requires make tidy)

tracking in #2814

Reply:
fixed in 0d573f27371e39f063829348bf239f7fba9916b2